webtau-groovy/src/main/groovy/com/twosigma/webtau/cfg/WebTauGroovyFileConfigHandler.groovy

```diff
@@ -44,25 +52,40 @@ class WebTauGroovyFileConfigHandler implements WebTauConfigHandler {
 
         validateEnv(cfg, workingDir, configPath)
 
-        def groovy = GroovyRunner.createWithoutDelegating(cfg.workingDir)
-
-        ConfigSlurper configSlurper = new ConfigSlurper(cfg.env)
-        def configScript = groovy.createScript(configPath.toUri().toString(), new ConfigBinding())
-
-        def parsedConfig = configSlurper.parse(configScript)
-        cfg.acceptConfigValues("config file", convertConfigToMap(parsedConfig))
+        ConfigObject parsedConfig = parseConfig(cfg, configPath)
 
         if (!parsedConfig) {
             return
         }
 
+        cfg.acceptConfigValues("config file", convertConfigToMap(parsedConfig))
+
         setupHttpHeaderProvider(parsedConfig)
         setupBrowserPageNavigationHandler(parsedConfig)
         setupReportGenerator(parsedConfig)
         setupPageElementGetSetValueHandlers(parsedConfig)
         setupTestListeners(parsedConfig)
     }
 
+    private static ConfigObject parseConfig(WebTauConfig cfg, Path configPath) {
+        try {
+            def groovy = GroovyRunner.createWithoutDelegating(cfg.workingDir)
+
+            ConfigSlurper configSlurper = new ConfigSlurper(cfg.env)
+            def configScript = groovy.createScript(configPath.toUri().toString(), new ConfigBinding())
+
+            def parsedConfig = configSlurper.parse(configScript)
+            return parsedConfig
+        } catch (Throwable e) {
+            if (ignoreConfigErrors.get()) {
+                ConsoleOutputs.err(StackTraceUtils.fullCauseMessage(e))
+                return null
```

Comment:
how does a `null` help the caller? I guess I don't fully understand the need/concept behind ignoring config errors.

Reply:
See above

Reply:
I shall leave the comment in the code